.evergreen/attach.yml

```diff
@@ -1,4 +1,18 @@
 functions:
+  attach-analytics-visualizer:
+    command: s3.put
+    type: system
+    params:
+      aws_key: ${AWS_ACCESS_KEY_ID}
+      aws_secret: ${AWS_SECRET_ACCESS_KEY}
+      aws_session_token: ${AWS_SESSION_TOKEN}
+      local_file: "ui/${app_dir}/dist/analytics.html"
+      remote_file: ${build_variant}/${task_id}/${execution}/analytics.html
```

Comment:
I wonder if we could upload this to some specific destination for mainline commit merges (or deploys?). So that we always have a single source of truth to link people too.

Reply:
Check the description, we actually don't need to! It's included in the vite build so it will be uploaded to S3 with a deploy to spruce.mongodb.com/analytics.html or parsley.mongodb.com/analytics.html